polaris/hub/client.py

```diff
@@ -304,9 +310,18 @@ def get_dataset(
         Returns:
             A `Dataset` instance, if it exists.
         """
-        return self._get_dataset(owner, name, ArtifactSubtype.STANDARD.value, verify_checksum)
+        with ProgressIndicator(
+            start_msg="Fetching dataset...",
+            success_msg="Fetched dataset.",
+            error_msg="Failed to fetch dataset.",
+        ):
+            try:
+                return self._get_v1_dataset(owner, name, ArtifactSubtype.STANDARD.value, verify_checksum)
```

Comment:
The `artifact_type` param is typed as `ArtifactSubtype`, but we pass a string in most places.

```suggestion
                return self._get_v1_dataset(owner, name, ArtifactSubtype.STANDARD, verify_checksum)
```